app/api/routers/breeze_buddy.py

```
@@ -702,3 +709,112 @@ async def get_call_details(
+                    headers={
+                        "Content-Disposition": f'attachment; filename="recording_{call_sid}.mp3"'
+                    }
+                )
+
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"Error fetching call recording: {e}", exc_info=True)
+        raise HTTPException(
+            status_code=500,
+            detail=f"Internal server error while fetching recording: {str(e)}"
```

Ruff (0.14.8)

Use explicit conversion flag

Replace with conversion flag

(RUF010)